### ISSUE TITLE
Inject MySQL Connection using v4

### ISSUE THREAD

**Issue by eldad87** [volatiletech/sqlboiler#744]
Hi,
I just switched from dep to mod and now I'm forced to use v4. With this new version I can't re-use and existing DB connections, instead I'm forced to use `driver.Assemble` method in order to orchestrate a DB connection.
It works, but the flexibility I had is now gone. For example, I'm no longer able to instrument my DB connection using OpenTracing like described on [this](https://ceshihao.github.io/2018/11/29/tracing-db-operations/) page and implemented on [this](https://github.com/eldad87/go-boilerplate/blob/master/src/cmd/grpc/app.go#L164) hobby project of mine.

Please advise,
Thanks!

**Comment by aarondl** [comment]
There should be no forcing. You can tell your go.mod file to use an older tag even though it's incompatible?

```driver.Assemble``` should not be called by user code ever. To create your connection simply call `sql.Open(...)` like in any other application you'd make and use that.

**Comment by eldad87** [comment]
Hi @aarondl ,

When I tried to use `sql.Open(...)`, I encountered an issue (doesn't apply to v3) with driver type conversion: `unable to update businesses row: sql: converting argument $4 type: driver ColumnConverter error converted uint to unsupported type uint64`.
I tried to check the [tests](https://github.com/volatiletech/sqlboiler/blob/master/drivers/sqlboiler-mysql/driver/mysql_test.go#L63), to see if something changed. 

I'll open a different issue about this subject.
